packages/client/src/components/Location/LocationFinder.tsx

```diff
@@ -66,7 +70,7 @@ const LocationFinder: FunctionComponent<LocationFinderProps> = ({
 
   const variables = {
     extraHouseNumberFull: "",
-    houseNumberFull: houseNumberFull,
+    houseNumberFull: transformHousenumberToValid(houseNumberFull),
```

Comment:
When starting localhost I get an error `Type 'undefined' is not assignable to type 'string'.`  Don't you?